.circleci/config.yml

```diff
@@ -200,25 +200,25 @@ jobs:
           command: |
             cd samples/java-valueentity-shopping-cart
             echo "Running mvn with SDK version: '$SDK_VERSION'"
-            mvn -Dakkaserverless-sdk.version=$SDK_VERSION -Dakkaserverless-maven-plugin.version=$SDK_VERSION verify -Pit
+            mvn versions:set -DnewVersion=$SDK_VERSION verify -Pit
```

Comment:
because the samples now use ${project.version} as the sdk version, it suffice to call `mvn versions:set`.

Reply:
OK, so this works because we have `<akkaserverless-sdk.version>${project.version}</akkaserverless-sdk.version>` - which is perhaps OK for us, but would definitely be different for users.

I'm OK with it, but TBQH I don't see the advantage of using `versions:set` over `-D` for the samples. We're not publishing those, are we?

Reply:
No, we are not publishing. But we do link them from the samples in the console. And users may checkout such a sample and run locally. In that case, we want the version to more or less aligned. 

Also, if we use `mvn versions:set` we can from the script update the samples to run them locally for our own tests. Otherwise, we need to pass -D to maven, but then, if you want to open such a sample in intellij, you want it to have a fixed sdk version, not something to change with -D. 

> but would definitely be different for users.

That won't be different for users since this is just a sample. When a user starts a new project from the archetype, they get the sdk version fixed and independent from their project version.

Reply:
> we want the version to more or less aligned.

We want the SDK version in the sample to be the latest, agreed!

> if you want to open such a sample in intellij, you want it to have a fixed sdk version, not something to change with -D

I can see how that's useful. Using -D means the `pom.xml` don't have to change all the time (marking the git repo dirty), but I guess there's no easy way to have both advantages ;)

> That won't be different for users since this is just a sample.

I meant, it's a bit confusing that the sample is different from what the user actually gets when they start a new project from the archetype. Not too big of a deal though.

Reply:
> it's a bit confusing that the sample is different from what the user actually gets when they start a new project from the archetype

Yeah, it's true.
We can try to improve it. We can probably do some `sed` on the pom and fix the version from that same script.